fix(docs): Fix nightly version builds

critesjosh opened the pull request:
There was a broken url between versions that was causeing the build to fail. This should fix it.

alejoamiras reviewed (approved):
Weird that this broke it. Should be a warning as far as I am aware. But let's gou!

sklppy88 commented on docs/docs/try_testnet.md:
```diff
@@ -21,7 +21,7 @@ description: "Connect to Aztec Alpha Testnet, explore the ecosystem, and start b
 - Try the [Aztec Starter Github repo](https://github.com/AztecProtocol/aztec-starter)
 - Follow our [tutorials](./developers/docs/tutorials/contract_tutorials/counter_contract.md) in order to write your first contract, deploy it, and interact with it using Aztec CLI and Aztec.js
-- If you are interested in connecting to devnet, please refer to [this page](/devnet/developers/getting_started_on_devnet).
+- If you are interested in connecting to devnet, please refer to [this page](https://docs.aztec.netowrk/devnet/developers/getting_started_on_devnet).
```
Why did it have to be changed like this ? Also seems like a typo

critesjosh replied:
Yes its a typo, good catch. Im going to investigate further as to why this isn't caught in our local builds

critesjosh commented:
I figured out the root of the issue. The versions included in the bulid were based off of the indices in the versions.json array. I've updated the inclusion logic to parse the version tags so it is more robust.

critesjosh committed:
fix(docs): Fix nightly version builds

There was a broken url between versions that was causeing the build to fail. This should fix it.

Co-authored-by: Josh Crites <jc@joshcrites.com>
